src/validation.cpp

```diff
@@ -4185,21 +4185,6 @@ bool LoadBlockIndex(const CChainParams& chainparams)
     return true;
 }
 
-static bool AddGenesisBlock(const CChainParams& chainparams, const CBlock& block, CValidationState& state)
```

Comment:
this should be reverted, see previous comment

Reply:
see https://github.com/dashpay/dash/pull/3096/commits/29b5f382d6550f9a244fd1abe2d213a1c75cb1ff